product_crd.yml

```diff
@@ -1,9 +1,47 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: products.k8sinitiative.3scale.net
 spec:
   group: k8sinitiative.3scale.net
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+          openAPIV3Schema:
+            type: object
+            properties:
+              spec:
+                type: object
+                properties:
+                  description: 
+                    type: string
+                  apps_count: 
+                    type: integer
+                  backends_count:
+                    type: integer
+                  id:
+                    type: integer
+                  name:
+                    type: string
+                  system_name:
+                    type: string
+                  link:
+                    type: string
+                  links:
```

Comment:
good to have them, not super necessary for the POC tho

Reply:
question: are you running a OAS3 validator?

Reply:
I'm not running any validator, but when you run `kubectl apply` it validates the schema
<img width="661" alt="Screenshot 2020-11-23 at 10 51 09" src="https://user-images.githubusercontent.com/13486237/99948505-dbfcc280-2d79-11eb-8dcd-c42689614bf2.png">